crates/wasm_interp/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 mod frame;
 mod instance;
+#[cfg(test)]
+mod tests;
+
```

Comment:
I originally removed this `mod tests` since the folder wouldn't exist during the nix build and it is not need outside of tests.

This caused
```
impl<'a, I: ImportDispatcher> Instance<'a, I> {
    #[cfg(test)]
    pub(crate) fn new<G>(
```

to have clippy errors since it was only used in the tests.